index.html

```diff
@@ -48,164 +54,206 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
         <img src="images/Normal Button/Disabled.png" alt="">
       </div>
   </section>
-  <section class="">
-    <div class="grid">
-
-      <div class="recent-works"><h3 class="hey-tex">My Recent Works</h3>
+  <section class="all-works">
+  
+      <div class="recent-works"><h3 class="hey-text">My Recent Works</h3>
       </div>
-        
-
+<div class="green"></div>
+      <div class="project-cards">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please style the layout of the work cards like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=FM3eIBvsAszGSaO5-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/221367043-fd6f47a0-858c-4212-b43a-94b152f64b7a.png)